samples/rtos/threadx-demo/main.cpp

```diff
@@ -3,98 +3,71 @@
  * This will make 4 threads and send data between them.
```

Comment:
Maybe at one point we could have a demo for each ThreadX class individually, so it's easy to learn about it one piece at a time. That doesn't need to be done now, but it's worth considering in the future

Reply:
I agree that would be nice in the future. When Ishan gets Renode automated testing working, he proposed making a set of samples that test individual objects/files functionality, so if that ever happens it would be a good opportunity to make separate samples.